src/apb/engine.py

```diff
@@ -632,3 +747,39 @@ def cmdrun_bootstrap(**kwargs):
         exit(1)
 
     print("Successfully bootstrapped Ansible Service Broker")
+
+
+def cmdrun_test(**kwargs):
+    project = kwargs['base_path']
+    spec = get_spec(project)
+
+    update_dockerfile(project)
+
+    if not kwargs['tag']:
+        tag = spec['image']
+    else:
+        tag = kwargs['tag']
+    print("Building APB using tag: [%s]" % tag)
+
+    client = docker.DockerClient(base_url='unix://var/run/docker.sock', version='auto')
+    client.images.build(path=project, tag=tag)
+
+    print("Successfully built APB image: %s" % tag)
+
+    # run image that was just created.
+    create_image_pod(tag)
+    test_result = retrieve_test_result()
+    test_results = []
+    if test_result is None:
+        print("Unable to retrieve test result.")
+        return
```

Comment:
Should this exit instead of return? I'm assuming this means it failed the test?

Reply:
As long as we call `clean_up_image_run` then yeah that makes sense to me. 

I want to make sure that we delete the things that we created so that they can re-run the `test` command without dealing with duplicate things, especially the pod. 
:+1: Will make that change